lib/yt/models/snippet.rb

```diff
@@ -51,6 +51,11 @@ def can_reply?
         @can_reply ||= data.fetch 'canReply', false
       end
 
+      def top_level_comment
+        top_level_comment = ActiveSupport::HashWithIndifferentAccess.new data['topLevelComment']
+        @top_level_comment ||= Yt::Comment.new top_level_comment
```

Comment:
Does a case exist where there is a commentThread without a topLevelComment?

On Mon, Apr 4, 2016 at 11:13 AM, Qiang Han <notifications@github.com> wrote:

> In lib/yt/models/snippet.rb
> <https://github.com/Fullscreen/yt/pull/271#discussion_r58420975>:
>
> > @@ -51,6 +51,11 @@ def can_reply?
> >          @can_reply ||= data.fetch 'canReply', false
> >        end
> >
> > +      def top_level_comment
> > +        top_level_comment = ActiveSupport::HashWithIndifferentAccess.new data['topLevelComment']
> > +        @top_level_comment ||= Yt::Comment.new top_level_comment
>
> Only if data['topLevelComment'] returns a ruby hash.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/Fullscreen/yt/pull/271/files/1b8f4211551179d6e52dae689f656bc965d29006#r58420975>
>



-- 


 Claudio Baccigalupo  | Manager, Software Engineering  |  Fullscreen, Inc.
<http://www.fullscreen.com>